packages/ketcher-core/src/domain/entities/struct.ts

```diff
@@ -288,14 +288,15 @@ export class Struct {
 
     const sgroupIdMap = {};
     this.sgroups.forEach((sg, sgroupId) => {
-      if (sg.atoms.some((aid) => !atomSet!.has(aid))) return;
+      const notInGroup = sg.atoms.some((aid) => !atomSet!.has(aid));
+      const atomsInSet = sg.atoms.filter((aid) => atomSet!.has(aid));
+      if (!atomsInSet.length && notInGroup) return;
```

Comment:
Maybe I'm missing something here but why do you need `notInGroup`? Conditions in `.some` and `.filter` are mutually exclusive so if there are no elements satisfying the `atomSet.has(aid)` condition, there is definitely at least one element which satisfies the inverse `!atomSet.has(aid)` condition

Reply:
Sometimes (when we copy a attachment point label) we are not in the group, but we need still filter some atoms,
but if we are not in the group and there are no atoms, we exit. Also, we must support the previous realization of merge info. 

![Screenshot from 2024-07-01 21-51-11](https://github.com/epam/ketcher/assets/117785539/ab75e555-86fb-4bee-b898-fbe78cf471c0)
![Screenshot from 2024-07-01 21-49-11](https://github.com/epam/ketcher/assets/117785539/20047c0a-0d27-4666-ad8f-8244d151f88f)

Perhaps there is another way to solve the problem.